### PR TITLE
Tuesday

### PR DESCRIPTION
**Pull request by generatorOf**
hm noch ein Comment? ich habe doch grade einen ausführlichen Kommentar geschrieben zum letzten commit...
hier ist die Kopie:
" added some minimals to additional-notes, added a starter greeting to README, transferred the corresponding content from documentation/additional-notes to appropriate wiki files some bash, some git, some markdown wiki, less pyenv and pynput for today. enjoy!

"

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*/
```
Comment (schwittlick) [inline .gitignore]: was genau ignorierst du damit z.b.?

File: additional-notes.md [hunk 2]
```diff
@@ -141,22 +143,49 @@ hm, here is the [usage of the with-statement](https://www.pythonforbeginners.com
...
-
+### new ideas:
+- [ ]  how to write an bash script, wich would open this file,
+write the todays date in the same format on the next cell of the table and
+put all unresolved tops from below tops to the entry of this day?
+I am thinking about an script for .bashrc :thinking:
+I know already we can use this:
+```
+$ date "+%F, %A %%%V" >> do.txt
+# 2020-01-14, Tuesday %03
```
Comment (schwittlick) [inline additional-notes.md]: what do you think about writing a python script that would do that?

File: daily-log.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<br> [x]   
+
+ideas: how to write an bash script, wich would open this file,
+write the todays date in the same format on the next cell of the table and
+put all unresolved tops from below tops to the entry of this day?
+I am thinking about an script for .bashrc :thinking:
+I know already we can use this:
+```
+$ date "+%F, %A %%%V" >> do.txt
+# 2020-01-14, Tuesday %03
```
Comment (schwittlick) [inline daily-log.md]: i'm reading these lines very often in your documents. https://en.wikipedia.org/wiki/Don%27t_repeat_yourself ;-)

File: git-wiki.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+1. `git push`
+hat das ganze dann initialisiert
+
+  * durch hinzufügen zum .bashrc folgender Zeilen
+```
+function lgit() {
+    git add .
+    git commit -a -m "$1"
+    git push
+}
```
Comment (schwittlick) [inline git-wiki.md]: keep this in your bash-wiki only, as a reference on how to write a bash functions. you shouldn't use this function in your daily git workflow though, so better remove it from here.

File: pyenv-wiki.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+#### pyenv
+	nach der pyenv Installation (`$ curl https://pyenv.run | bash`):
+`pyenv install x.y.z` zum installieren von einer anderen Python version.
+```
+pyenv install 3.6.8
+pyenv virtualenv 3.6.8 cursor
+pyenv activate cursor
+pip install -r requirements.txt
+./install_and_run.sh
```
Comment (schwittlick) [inline pyenv-wiki.md]: this line is not pyenv-specific, better remove it from here.

how do you deactivate a pyenv? how do you check which version your pyenv is? you might need these commands once in a while and they should be documented here

File: git-wiki.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+________
+
+### workflow with every new project to observe:
+* git clone repository
+* create a directory
+* create an pyenv version for this project
+* activate it -->
+`(name) name@pc:$`
+* try out with pyCharm - create files
+* run from terminal over pyenv
```
Comment (schwittlick) [inline git-wiki.md]: this is not git-only specific, try to find a better place to keep this. still document the "git clone" command in this document.

File: git-wiki.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+* mit `git branch v2` habe ich einen neuen Branch "v2" angelegt
+* mit `git checkout v2` switche ich in den neuen Branch
+* with `git push --set-upstream origin v2` we created origin space on github for our branch
+* then I commited and pushed the content to the new branch space
+* then I pulled request with an button on github
+* then we descussed the changes in the chat on "pull request" segment on the github
+* then we merged the branch with the master branch, closed the issue,
+linked the closed pull request to the closed issue
+* it will be the future workflow
+* to erase the local branch use: ` git branch -d branch-name`
```
Comment (schwittlick) [inline git-wiki.md]: 👌

File: additional-notes.md [hunk 8]
```diff
@@ -94,26 +96,26 @@ ________
...
+- [x] use pynput over an certain python version in pyenv 
+- [x] on troubles as first - google it
+- [x] use less commits but with meaningfull comments (!)
+- [x] create a branch using method described on [friday](#git-branch)
+- [x] integrate pyenv to my pyCharm
 https://realpython.com/intro-to-pyenv/
 - [ ] read about pip
 - [ ] create  pyenv for pynput 
-- [ ] google how to pyenv in pyCharm integrate
+- [x] google how to pyenv in pyCharm integrate
```
Comment (schwittlick) [inline additional-notes.md]: you managed to make this work? please document the steps

File: .gitignore (hunk 1 above)
Comment (generatorOf) [reply .gitignore]: directories/ ordner

Comment (generatorOf) [reply .gitignore]: dachte ich, jedenfalls für meine 2  Ordner "cursor" und "pynput" hat es funktioniert

File: additional-notes.md (hunk 2 above)
Comment (generatorOf) [reply additional-notes.md]: we will see

File: daily-log.md (hunk 3 above)
Comment (generatorOf) [reply daily-log.md]: ok, its becouse of the restructuring, I am not sure if this part better belongs to the short version or to the long one.
for me it is a reminder, and it doesn't disturb anywhere, but after time we will see what has where the best place.

File: git-wiki.md (hunk 4 above)
Comment (generatorOf) [reply git-wiki.md]: I see

File: pyenv-wiki.md (hunk 5 above)
Comment (generatorOf) [reply pyenv-wiki.md]: I know, I just copied the parts to the corresponding files from the main documentation, I want still to change a lot in each separate file

File: git-wiki.md (hunk 6 above)
Comment (generatorOf) [reply git-wiki.md]: ok

File: additional-notes.md (hunk 8 above)
Comment (generatorOf) [reply additional-notes.md]: ok, I will look up for it

File: git-wiki.md (hunk 6 above)
Comment (generatorOf) [reply git-wiki.md]: I'll ban it to the "additional-notes.md"

Comment (generatorOf) [reply git-wiki.md]: v

File: daily-log.md (hunk 3 above)
Comment (generatorOf) [reply daily-log.md]: v

File: git-wiki.md (hunk 4 above)
Comment (generatorOf) [reply git-wiki.md]: v

File: do.txt [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+2020-01-14, Tuesday
+2020-01-14, Tuesday
+###
+2020-01-14, Tuesday
+### + date +%F, %A
+### = date +%F, %A
+2020-01-14, Tuesday 03
+2020-01-14, Tuesday % 03
+2020-01-14, Tuesday %03
+date "+%F, %A %%%V" >> do.txt
```
Comment (schwittlick) [inline do.txt]: git rm do.txt
git commit -m "removes do.txt"
git push origin tuesday

File: .gitignore (hunk 1 above)
Comment (schwittlick) [reply .gitignore]: resolved in this commit: https://github.com/generatorOf/internship/pull/5/commits/7b5297617e22ad4d8c0482c7efe889aaca063cf9

**Pull request merged**